packages/angular/test/base/src/app/standalone/icon/icon.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { IonIcon } from '@ionic/angular/standalone';
+import { logoIonic, logoIonitron, logoApple, logoAndroid } from 'ionicons/icons';
+import { addIcons } from 'ionicons';
+
+addIcons({ logoIonic, logoIonitron, logoApple, logoAndroid });
```

Comment:
Played with this locally, everything looks/works good 👍 

It seems to work consistently if you have this outside the class or within the constructor. Does not work (too late in the execution cycle, after `connectedCallback` fires) if you try to add icons in the `ngOnInit` lifecycle.

Reply:
Good catch! We'll want to note this in the docs